mltsp/Flask/flask_app.py

```diff
@@ -41,7 +31,10 @@
 from sklearn.externals import joblib
 import numpy as np
 import yaml
-from flask.ext import stormpath
+if os.getenv("flask_testing") == "True":
```

Comment:
Traditionally, environment variables are all caps, and take a numeric value.  So, how about:

```
if os.getenv('MLTSP_DEBUG') == "1":
```